yaks/README.md

```diff
@@ -552,6 +552,28 @@ yaks = Yaks.new do
 end
 ```
 
+Yaks also has support for respecting the `include` query parameter (e.g.
+`include=posts.comments,account`), which is a behaviour you can include in
+your mappers:
+
+```ruby
+require "yaks/behaviour/optional_includes"
+
+class UserMapper < Yaks::Mapper
+  include Yaks::Behaviour::OptionalIncludes
+
+  has_many :posts
+  has_one :account
+end
+
+# ...
+
```

Comment:
Actaully, I meant to say that from the top-level it works, but it throws a lot of offenses unrelated to my files, so it doesn't seem like the same Rake task that is run on Travis.